### PR TITLE
Added SVTs for ContainerInstances

### PR DESCRIPTION
**Pull request by gamitt-msft**
## Description
</br>
 Brief description of the requested change.

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK/Framework/Configurations/SVT/Services/ContainerInstances.json [hunk 1]
```diff
@@ -36,6 +36,40 @@
...
+    {
+      "ControlID": "Azure_ContainerInstances_AuthZ_Containers_Trust",
+      "Description": "A container group must contain only containers which trust each other",
+      "Id": "ContainerInstances140",
+      "ControlSeverity": "High",
+      "Enabled": true,
+      "Automated": "Yes",
+      "MethodName": "CheckContainerTrust",
+      "Rationale": "Container hosted on same container group can monitor traffic of other containers and access the file system of host OS. Hence the same container group must not host containers which do not trust each other.",
+      "Recommendation": "Ensure that the containers in a container group trust each other.",
```
Comment (sbyna-msft) [inline src/AzSK/Framework/Configurations/SVT/Services/ContainerInstances.json]: steps to validate on how team can verify the container groups

File: src/AzSK/Framework/Configurations/SVT/Services/ContainerInstances.json [hunk 2]
```diff
@@ -36,6 +36,40 @@
...
+    {
+      "ControlID": "Azure_ContainerInstances_DP_Review_Registry",
+      "Description": "Make sure container images are hosted on a trustworthy registry having necessary authentication, authorization and encryption mechanisms",
+      "Id": "ContainerInstances130",
+      "ControlSeverity": "High",
+      "Enabled": true,
+      "Automated": "Yes",
+      "MethodName": "CheckRegistry",
+      "Rationale": "If a container image is hosted on an untrusted registry, it can compromise application code and data.",
+      "Recommendation": "Ensure that the registry, which hosts the image, is trustworthy. Review registry configurations carefully for any misconfigurations related to authentication, authorization and encryption.",
```
Comment (sbyna-msft) [inline src/AzSK/Framework/Configurations/SVT/Services/ContainerInstances.json]: We need to provide instructions or article on how to secure the registry. like if the status is verify what should id do?

File: src/AzSK/AzSKStaging.psd1 [hunk 3]
```diff
@@ -12,7 +12,7 @@
     RootModule             = '.\AzSK.psm1'
 
     # Version number of this module.
-    ModuleVersion          = '2.0.0.0'
+    ModuleVersion          = '3.2.0.0'
```
Comment (VishalHaibatpure) [inline src/AzSK/AzSKStaging.psd1]: We can not change default version here. Its been used in deployment pipeline

Comment (gamitt-msft) [reply src/AzSK/AzSKStaging.psd1]: I rollbacked it.

**Pull request merged**